templates/docs/settings/spacing-settings.md

```diff
@@ -7,70 +7,60 @@ context:
 # Spacing
 
 <hr>
+Spacing in Vanilla is controlled via a set of variables. There are two kinds of variables - nudges, which keep text aligned to the baseline grid, and multiples of the spacing unit, which define vertical and horizontal spacing.
 
-Vanilla uses numerous spacing variables across the codebase in order to ensure consistency in spacing between components, and to ensure typographic elements sit on the baseline grid.
+## The baseline grid
 
-## Spacing unit
+Everything in Vanilla adheres to a `.5rem` baseline grid. This is achieved by ensuring that:
 
-Vanilla uses a default spacing unit of `.5rem` (`8px`) as a basis to calculate spacing inside and between components, as well as the line-heights of the different type sizes.
+- text baselines are "nudged" until they fall precisely on the grid
+- any margin, padding or other positioning happens in multiples of the baseline grid spacing unit.
```

Comment:
I'd move this up above the point with nudges.

Reply:
The nudges is what makes vertical rhythm work, I don't want them to feel like an optional final step - that's why I always put it at the top.